bosi/rhosp_resources/build_scripts/rhosp_packager.sh

```diff
@@ -2,16 +2,45 @@
 
 # Following build params expected for this script:
 # OpenStackBranch
-# BosiBranch
-# RHOSPVersion
+# BcfBranch
 # Revision
+# IvsBranch (optional)
+
+# mapping for OpenStackBranch to RHOSPVersion, default is latest = 9
+# occasionally cleanup when we stop supporting certain versions
+RHOSPVersion="9"
+case "$OpenStackBranch" in
+  *"mitaka"*) RHOSPVersion="9" ;;
+  *"liberty"*) RHOSPVersion="8" ;;
+  *"kilo"*) RHOSPVersion="7" ;;
+esac
+
+# if IvsBranch is not specified, it is same as BcfBranch
+if [ -z  $IvsBranch  ]
+then
+    IvsBranch="$BcfBranch"
+fi
+
+# if branch is not master, append 'v' e.g. v3.7.0
+if [ $IvsBranch != "master" ]
```

Comment:
> IVS version should be the same as BCF version, no need for the extra configs. (if workflow is different, we need to make sure IVS adopts what BCF doesn)

so we can update the build scripts for all the other packages and then update this one to not have any special casing. right now, for beta, i don't think it would be feasible to change those before we release :)